README.md

```diff
@@ -1,4 +1,26 @@
 # verejne.digital
 Projekt [verejne.digital](https://verejne.digital?about) má za cieľ aplikovať umelú inteligenciu na dáta zverejňované slovenskými verejnými inštitúciami.
 
-TODO: Pridať dokumentáciu ako developovať frontend, backend, ako vytvárať nový projekt, a kde sú dáta.
+## Developovanie
+Kód je roztrúsený vo viacrých priečinkoch (static, obstaravania, prepojenia, profil...). 
+
+### Frontend
+
+#### static
+Ḱód [https://verejne.digital/](https://verejne.digital/), naprogramovaný v obyčajnom JS.
+
+#### obstaravania
+Kód [https://verejne.digital/obstaravania/index.html](https://verejne.digital/obstaravania/index.html), automaticky generovaný python skriptom [main.py](https://github.com/verejnedigital/verejne.digital/blob/master/obstaravania/main.py)
+
+#### prepojenia
+Kód [https://verejne.digital/prepojenia/index.html](https://verejne.digital/prepojenia/index.html).
+V priečinku `client` sa nachádza kód frontendu. Zvyšok priečinka tvorí backend.
+
+#### profil
+Kód [https://verejne.digital/profil/](https://verejne.digital/profil/).
+V priečinku `client` sa nachádza kód frontendu.
+
+Na developovanie react-u treba mať nainštalovaný `Node.js` a `npm`. Pred spustením, treba nainštalovať potrebné balíčky: `npm i`. Frontend spustíte pomocou `npm run start`.
```

Comment:
preco nie yarn? Zatial to vyzera byt npm vs yarn agnostic, preco nezobrat lepsiu technologiu?